controllers/topics/getLatestPost.js

```diff
@@ -1,8 +1,10 @@
 const {QueryTypes} = require('sequelize');
 const {sequelize, Topics} = require('../../databases/models');
+const {StreamChat} = require('stream-chat');
 
 module.exports = async (req, res) => {
   const {name} = req.query;
+  const {userId, token} = req;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The StreamChat library is imported and used without any error handling for potential import errors. This could lead to unhandled exceptions if the library fails to load. Consider wrapping the import in a try-catch block or ensure that your package.json file correctly lists `stream-chat` as a dependency.



<!-- This is an auto-generated comment by OSS CodeRabbit -->